### DISCUSSION TITLE
What are your infill settings?

### DISCUSSION THREAD

**Discussion by cryptohazey** [aderusha/DDD-Printable-Wall-Control-System#17] [Q&A] [answered]
First time printing these. What are your infill settings? Guessing it would matter for strength?

**Comment by aderusha** [comment, marked as answer]
More perimeters == more better.  In my experience, if the hook fails due to load it will fail somewhere around here:
![image](https://user-images.githubusercontent.com/9554705/137098230-17ba57da-24e6-406c-bdbf-bf7612b46ad6.png)

So, goal is to make sure the hook has as much "meat" as possible.  I'm using PrusaSlicer here.  I can add a "modifier" slab and set it to the height of the hook (they are 2.2mm thick), then set perimeters to a high value (say, something like 15).  Now, my first 2.2mm will print like this:
![image](https://user-images.githubusercontent.com/9554705/137098854-e597b6ea-03f6-494f-9bbc-90b72b9c97e6.png)

Then, the rest of the part can print with some reasonable value for infill.  I still like to crank perimeters for the rest of the print, but only to maybe 4 or so, with 40% infill to make it nice and beefy.
